Tests/UnitTests/Networking/Backend/BackendPostReceiptDataTests.swift

```diff
@@ -715,13 +715,17 @@ class BackendPostReceiptDataTests: BaseBackendPostReceiptDataTests {
         expect(completionCalled.value).toEventually(equal(2))
     }
 
-    func testGetsEntitlementsWithVerifiedResponse() {
```

Comment:
The diff here is horribly hard to read for no reason.
I simply moved 2 tests into a separate class, which is this:
```swift
class BackendPostReceiptWithSignatureVerificationTests: BaseBackendPostReceiptDataTests {

    override var verificationMode: Configuration.EntitlementVerificationMode { .informational }

    func testGetsEntitlementsWithVerifiedResponse() {
        self.httpClient.mock(
            requestPath: .postReceiptData,
            response: .init(statusCode: .success,
                            response: Self.validCustomerResponse,
                            verificationResult: .verified)
        )

        let result = waitUntilValue { completed in
            self.backend.post(receiptData: Self.receiptData,
                              productData: nil,
                              transactionData: .init(
                                 appUserID: Self.userID,
                                 presentedOfferingID: nil,
                                 unsyncedAttributes: nil,
                                 storefront: nil,
                                 source: .init(isRestore: false, initiationSource: .purchase)
                              ),
                              observerMode: false,
                              completion: { result in
                completed(result)
            })
        }

        expect(result).to(beSuccess())

        if #available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *) {
            expect(result?.value?.entitlements.verification) == .verified
        }
    }

    func testGetsEntitlementsWithFailedVerification() {
        self.httpClient.mock(
            requestPath: .postReceiptData,
            response: .init(statusCode: .success,
                            response: Self.validCustomerResponse,
                            verificationResult: .failed)
        )

        let result = waitUntilValue { completed in
            self.backend.post(receiptData: Self.receiptData2,
                              productData: nil,
                              transactionData: .init(
                                 appUserID: Self.userID,
                                 presentedOfferingID: nil,
                                 unsyncedAttributes: nil,
                                 storefront: nil,
                                 source: .init(isRestore: false, initiationSource: .purchase)
                              ),
                              observerMode: false,
                              completion: { result in
                completed(result)
            })
        }

        expect(result).to(beSuccess())

        if #available(iOS 13.0, macOS 10.15, tvOS 13.0, watchOS 6.2, *) {
            expect(result?.value?.entitlements.verification) == .failed
        }
    }

}
```

Reply:
thanks for the heads up!